pcsx2-qt/MainWindow.cpp

```diff
@@ -106,6 +106,17 @@ MainWindow::MainWindow()
 
 #if !defined(_WIN32) && !defined(__APPLE__)
 	s_use_central_widget = DisplayContainer::isRunningOnWayland();
+
+    /*  native window rendering is broken in wayland.
+     *  Let's workaround it by disabling it for every widget besides
+     *  DisplayWidget.
+     *  QTBUG-133919, reported upstream by govanify 
+     */
+    if(DisplayContainer::isRunningOnWayland())
+    {
+        QGuiApplication::setAttribute(Qt::AA_NativeWindows, false);
+        QGuiApplication::setAttribute(Qt::AA_DontCreateNativeWidgetSiblings, true);
+    }
```

Comment:
Formatting, tabs instead of spaces, also space after if.

Reply:
done, sorry for the mess up, did the PR in a vm to test it out!